src/popup/router/pages/Retip.vue

```diff
@@ -40,14 +40,14 @@ import { SCHEMA } from '@aeternity/aepp-sdk';
 import { MAGNITUDE, calculateFee } from '../../utils/constants';
 import { convertToken } from '../../utils/helper';
 import deeplinkApi from '../../../mixins/deeplinkApi';
-import AmountInput from '../components/AmountInput';
+import InputAmount from '../components/InputAmount';
```

Comment:
Why did we do it this way around? It sounds wrong to me tbh, usually we put the description first and then the type. Aka `mainButton`, `navLink` (...) so I would expect this to be `AmountInput`.